### PR TITLE
fix get user cap

### PR DESCRIPTION
**Pull request by edwardEvans094**

### CODE REVIEW

File: src/js/services/account.js [hunk 1]
```diff
@@ -66,22 +66,22 @@ export default class Account {
...
-        })
-        .catch((err) => {
-          console.log(err)
-          acc.maxCap = "infinity"
-          acc.rich = false          
-          resolve(acc)
-        })
-      })
-    })
+    // promise = promise.then((acc) => {
```
Comment (andrewsource147) [inline src/js/services/account.js]: Please remove it

File: src/js/sagas/globalActions.js [hunk 2]
```diff
@@ -1,9 +1,15 @@
 import { put, call, takeEvery } from 'redux-saga/effects'
+
 import * as actions from '../actions/globalActions'
+import * as exchangeActions from '../actions/exchangeActions'
+import * as transferActions from '../actions/transferActions'
+import * as limitOrdersActions from '../actions/limitOrderActions'
```
Comment (andrewsource147) [inline src/js/sagas/globalActions.js]: Please remove ti

File: src/js/sagas/globalActions.js [hunk 3]
```diff
@@ -185,12 +191,34 @@ export function* changelanguage(action) {
...
+export function* checkUserEligible(action) {  
+  try{
+    var {ethereum} = action.payload
+    var state = store.getState()
+    var account = state.account.account
+    var address = account.address
+    var result = yield call([ethereum, ethereum.call], "getUserMaxCap", address)
+
+    if(result.success && result.eligible){
+      yield put(limitOrdersActions.clearErrorEligible())
```
Comment (andrewsource147) [inline src/js/sagas/globalActions.js]: Why limitOrderActrions need to clear error?

File: src/js/services/account.js (hunk 1 above)
Comment (edwardEvans094) [reply src/js/services/account.js]: removed

File: src/js/sagas/globalActions.js (hunk 2 above)
Comment (edwardEvans094) [reply src/js/sagas/globalActions.js]: removed

**Pull request merged**